x-pack/plugins/security_solution/cypress/tasks/alerts_detection_rules.ts

```diff
@@ -119,6 +121,32 @@ export const waitForRuleToBeActivated = () => {
 };
 
 export const waitForRulesToBeLoaded = () => {
-  cy.get(LOADING_SPINNER).should('exist');
-  cy.get(LOADING_SPINNER).should('not.exist');
+  cy.get(ASYNC_LOADING_PROGRESS).should('exist');
+  cy.get(ASYNC_LOADING_PROGRESS).should('not.exist');
+};
+
+// when using, ensure you've called cy.clock prior in test
```

Comment:
Is this comment still valid? Didn't see a cy.clock anywhere?

Reply:
Deleted in followup https://github.com/elastic/kibana/pull/83023